components/gallery/GalleryItemAction/GalleryItemActionType/GalleryItemBuy.vue

```diff
@@ -47,4 +47,28 @@ const actionRef = ref(null)
 onClickOutside(actionRef, () => (active.value = false))
 </script>
 
-<style lang="scss" scoped></style>
+<style lang="scss" scoped>
+@import '@/styles/abstracts/variables';
+
+.dark-mode {
+  .slide {
+    box-shadow: 4px 4px 0 white;
+  }
+
+  .is-neo {
+    background: $k-accent;
+    color: black;
+
+    &:hover {
+      background: $k-dark;
+      color: white;
+    }
+  }
+}
+
+.is-neo {
+  &:hover {
+    background: white;
+  }
+}
```

Comment:
shouldn't this be done in `libs/ui` as I don't see `is-neo` class being used here, it must be an inherited class